.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.tt]
+insert_final_newline = false
```

Comment:
Are these based on our coding standards from our public wiki? Please confirm

Reply:
> Are these based on our coding standards from our public wiki? Please confirm

It doesn't look like it will contradict any of out [Coding-Standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)